frontend/ui/widget/filechooser.lua

```diff
@@ -559,14 +559,20 @@ end
 
 -- Used in ReaderStatus:onOpenNextDocumentInFolder().
 function FileChooser:getNextFile(curr_file)
+    local show_finished = FileChooser.show_finished
+    FileChooser.show_finished = true
```

Comment:
Shouldn't this be *false* if we want to always ignore finished books here?

Reply:
Oh, okay, I get it.

We need it set to actually collect finished books, but the skipping is handled manually later, instead of relying on show_file ;).

Reply:
Exactly. The user just has marked the book as finished, but we must keep it in the item_table to find the next one in accordance with the sorting order.